smf/sled-agent/config-rss.toml

```diff
@@ -11,6 +11,22 @@ rack_subnet = "fd00:1122:3344:0100::"
 # For values less than 2, no rack secret will be generated.
 rack_secret_threshold = 1
 
+[gateway]
+
+# IP address of Internet gateway
+#
+# NOTE: In the lab, use "172.20.15.225"
+# address = "192.168.1.1"
+
+# MAC address of the internet gateway in the local network, i.e., of the above
+# IP address.
+#
+# NOTE: This is currently configured for the lab. Developers should override
+# this with whatever value they wish to provide inbound connectivity to guests
+# in their local network, using the current workaround methods in OPTE. See
+# how-to-run.adoc for details on how to determine the value for your network.
+mac = "00:0d:b9:54:fe:e4"
```

Comment:
I'm not sure if this is a new change, but does it make sense to have the `mac` specified but not the IP address? Both would be required to actually launch guests with inbound connectivity, so it seems like we should have both or neither.

Reply:
FWIW, this format matches the config before this PR. However, obviously, it could still be improved.

https://github.com/oxidecomputer/omicron/blob/b270033cd460f4ae429d0f9643b0a92548b39719/sled-agent/src/opte/illumos/port_manager.rs#L228-L230 looks like the only "real" usage of the `mac` field.

https://github.com/oxidecomputer/omicron/blob/b270033cd460f4ae429d0f9643b0a92548b39719/sled-agent/src/services.rs#L393-L401 looks like the "real" usage of the `address` field.

As far as I can tell, the MAC usage is unconditional, but the gateway address looks conditional for adding a default route.

When the external IP workaround hack goes away, do you expect either of these to remain?

Reply:
(based on comments, it seems like the IP will be here to stay, but the MAC address would go away?)

Reply:
Yeah, I'd expect them both to go away. The IP address will ultimately be an external address from the customer's network. The MAC won't matter here -- the switch will need to know it, but Nexus doesn't.

Reply:
The switch itself may not even know the MAC address. For IP level connectivity we will never be asking customers about MAC addresses. The only exception may be for some particular routing config or allow/deny list / lldp.